revolv/templates/base/sign_in.html

```diff
@@ -55,9 +55,15 @@
             {% for error in signup_form.non_field_errors %}
               <small class="error non-field-error">{{ error }}</small>
             {% endfor %}
-            {% for signup_field in signup_form %}
-                {% include "base/partials/placeholder_formfield.html" with field=signup_field %}
-            {% endfor %}
+            {% include "base/partials/placeholder_formfield.html" with field=signup_form.username %}
+            {% include "base/partials/placeholder_formfield.html" with field=signup_form.password1 %}
+            {% include "base/partials/placeholder_formfield.html" with field=signup_form.password2 %}
+            {% include "base/partials/placeholder_formfield.html" with field=signup_form.first_name %}
+            {% include "base/partials/placeholder_formfield.html" with field=signup_form.last_name %}
+            {% include "base/partials/placeholder_formfield.html" with field=signup_form.email %}
```

Comment:
It's because the placeholder_formfield isn't suited correctly to handle the new boolean field of the sign up for email. Tbh I think it's okay to have a non-partial for the checkbox